styles/main.css

```diff
@@ -0,0 +1,178 @@
+* {
+  font-family: 'Space Mono', monospace;
+}
+
+h1 {
+  font-size: 48px;
+}
+
+h2 {
+  font-size: 28px;
+}
+
+p {
+  font-size: 14px;
+  line-height: 1.25em;
+}
+
+li {
+  list-style-type: none; 
+}
+
+ul {
+  padding-inline-start: 0px;
+  display: flex;
+  flex-flow: row;
+  justify-content: space-between;
+}
+
+img {
+  width: 200px;
+  height: auto;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-row: minmax(500px, auto);
+  padding: 30px;
+  justify-content: space-around;
+  align-items: center;
+  border: 1.5em solid;
+  border-image: linear-gradient(to top, blue, pink) 1;
```

Comment:
Love this gradient background! Learned something new, also your aesthetic is aspirational.